### PR TITLE
Added Location codes to SlimLocationInfo, removed countryCode from Ro…

### PR DESCRIPTION
**Pull request by hasmikharutyunyan**
…omContractSetAvailability, maken nullable remarks and polices in Deadline model

### CODE REVIEW

**Review by oleg1std — APPROVED** [review]

File: HappyTravel.EdoContracts/Accommodations/Internals/SlimLocationInfo.cs [hunk 1]
```diff
@@ -8,30 +8,38 @@ namespace HappyTravel.EdoContracts.Accommodations.Internals
...
-        public SlimLocationInfo(string address, string country, string locality, string localityZone, in GeoPoint coordinates)
+        public SlimLocationInfo(string address, string country, string countyCode, string locality, string localityCode, string localityZone,
+            string? localityZoneCode, in GeoPoint coordinates)
         {
             Address = address;
             Coordinates = coordinates;
             Country = country;
+            CountryCode = countyCode;
             Locality = locality;
+            LocalityCode = localityCode;
```
Comment (kirillta) [inline HappyTravel.EdoContracts/Accommodations/Internals/SlimLocationInfo.cs:19]: Why do we need those codes in the contracts? They are defined by supplier

Comment (hasmikharutyunyan) [reply HappyTravel.EdoContracts/Accommodations/Internals/SlimLocationInfo.cs:19]: I thought if we add countryCode, better to add other codes to. If this codes we will never use, I can remove.

File: HappyTravel.EdoContracts/Accommodations/Deadline.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ namespace HappyTravel.EdoContracts.Accommodations
     public readonly struct Deadline
     {
         [JsonConstructor]
-        public Deadline(DateTime? date, List<CancellationPolicy> policies, List<string> remarks, bool isFinal)
+        public Deadline(DateTime? date, List<CancellationPolicy>? policies = default, List<string>? remarks = default, bool isFinal = true)
```
[the remark this answers is not in the document]
Comment (hasmikharutyunyan) [reply HappyTravel.EdoContracts/Accommodations/Deadline.cs:14]: Yes, As I remember, we prefer `default` than `null`

File: HappyTravel.EdoContracts/Accommodations/Internals/SlimLocationInfo.cs (hunk 1 above)
Comment (kirillta) [reply HappyTravel.EdoContracts/Accommodations/Internals/SlimLocationInfo.cs:19]: Yes, please

**Review by kirillta — APPROVED** [review]

**Pull request merged**